recipes/x265/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set version = "2.9" %}
+
+package:
+  name: x265
+  version: {{ version }}
+
+source:
+  url: https://bitbucket.org/multicoreware/x265/downloads/x265_{{ version }}.tar.gz
+  md5: 693ee4ce7929a59735f441107310f0bb
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+    - pkgconfig              # [unix]
+  host:
+    - nasm
```

Comment:
I think `nasm` should go in the `build` section.

Reply:
done!